config/rewrite.yml

```diff
@@ -5,10 +5,17 @@ displayName: Auto Fixes
 description: List of auto fixes from different providers.
 recipeList:
   - org.checkstyle.autofix.CheckstyleAutoFix:
-      violationReportPath: "target/cs_errors.xml"
-      configurationPath: "config/checkstyle-checks.xml"
-      propertiesPath: "config/openrewrite-recipes-checkstyle.properties"
+      violationReportPath: target/cs_errors.xml
```

Comment:
Move this update to separate PR with commit prefix as minor to merge ASAP and make it clear that it just noticed, no tool involved to demand this